doc/user/content/concepts/indexes.md

```diff
@@ -42,5 +51,10 @@ indexes in another cluster.
 
 ## Related pages
 
+- [Optimization](/transform-data/optimization)
 - [Views](/concepts/views)
 - [`CREATE INDEX`](/sql/create-index)
+
+<style>
+red { color: Red; font-weight: 500; }
+</style>
```

Comment:
This should probably live under [`/doc/user/assets/sass/_content.scss`](https://github.com/MaterializeInc/materialize/blob/main/doc/user/assets/sass/_content.scss).

Reply:
I was actually thinking maybe adding the specific colors to https://github.com/MaterializeInc/materialize/blob/main/doc/user/assets/sass/main.scss  and having a shortcode file that list some text styles, referencing the colors from the main.scss.  But will know better as I work through the quickstart to get a sense of what all we might need.

Reply:
Can you tell I known nothing about CSS. And sounds good! Just trauma reaction from having random JS code blocks embedded in the markdown files at some point.